uvloop/handles/stream.pyx

```diff
@@ -1,3 +1,8 @@
+# MIT and Apache 2.0
+#
+# Copyright (c) 2021, NVIDIA CORPORATION.
```

Comment:
Not sure how noting copyright holders/contributors works here. If there is a preferred way to do this, please let me know and we can handle that way

Reply:
Where is this copyright coming from?

Reply:
Thank you for the contribution. This is the first time a contributor asks to add explicit copyright notices to a file and I don't think I like to add them everywhere, especially for minor contributions.

Would changing "Copyright (c) 2015-present MagicStack Inc.  http://magic.io" to "Copyright (C) 2016-present the uvloop authors and contributors." in our LICENSE files alleviate the need of an explicit copyright note you?

Reply:
Also: https://www.linuxfoundation.org/blog/copyright-notices-in-open-source-software-projects/

Reply:
Yeah I think that would help if that would be agreeable. Would you like to make that change or should I make that here?

Reply:
Have pushed the change as suggested to the PR. Please let me know what you think :)